### PR TITLE
Add update guide

### PR DESCRIPTION
**Pull request by natke**
Add a single page update guide, akin to the installation guide.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/65cc4117d6ca9ce998b52bdb119c24061e40fbd6/articles/install-guide/update.md) | 88 | 70 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f138106b-dec9-4129-9c08-1c9c2f2a30ba_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/dc27662876936f5c500562fbb1fa07b4d0d9d3c6/articles/install-guide/update.md) | 89 | 73 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8ab82989-2416-40d3-9092-7f62268c5648_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/bca75e1f155773fe6e6572f22a789aa87b60e51a/articles/install-guide/update.md) | 89 | 73 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/066c9a0a-d710-4dbf-9cf8-6b703b209abc_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

File: .gitignore [hunk 1]
```diff
@@ -10,4 +10,5 @@ tex2pdf.*/
 .openpublishing.buildcore.ps1
 
 *.orig
-.vs/
\ No newline at end of file
+.vs/
+.DS_Store
```
Comment (cgranade) [inline .gitignore]: Oh, `.DS_Store`.

File: articles/install-guide/update.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+ms.topic: article
+ms.custom: how-to
+uid: microsoft.quantum.update
+---
+
+# Update the Microsoft Quantum Development Kit (QDK)
+
+Learn how to update the Microsoft Quantum Development Kit (QDK) to the latest version.
+
+This article assumes that you already have the QDK installed. If you are installing for the first time, then please refer to the [installation guide](xref:microsoft.quantum.install)
```
Comment (cgranade) [inline articles/install-guide/update.md]:
```suggestion
This article assumes that you already have the QDK installed. If you are installing for the first time, then please refer to the [installation guide](xref:microsoft.quantum.install).
```

File: articles/install-guide/update.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+    pip install qsharp --upgrade
+    ```
+
+1. Verify the `qsharp` version
+
+    ```bash
+    pip show qsharp
+    ```
+
+    You see the following output:
```
Comment (cgranade) [inline articles/install-guide/update.md]:
```suggestion
    You should see the following output:
```

File: articles/install-guide/update.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+    > The project templates are updated with the extension. The updated templates apply to newly created projects only. The code for your existing projects is not updated when the extension is updated.
+
+1. Update the QDK packages
+
+    - Open an existing application
+    - Select **Dependencies** in the Solution Explorer
+    - Select **Manage NuGet Packages**
+    - Update the **Microsoft.Quantum** packages to the latest version
+
+1. You can now run your application with the latest QDK
```
Comment (cgranade) [inline articles/install-guide/update.md]:
```suggestion
1. You can now run your application with the latest QDK.
```

File: articles/install-guide/update.md [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+
+## C#, using VS Code
+
+1. Update the Quantum VS Code extension
+
+    - Restart VS Code
+    - Navigate to the **Extensions** tab
+    - Select the **Microsoft Quantum Development Kit for Visual Studio Code** extension
```
Comment (cgranade) [inline articles/install-guide/update.md]: These steps should happen automatically unless the user has opted out from VS Code's autoupdate functionality.

File: articles/install-guide/update.md [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+    - Reload the extension
+
+1. Update the Quantum project templates:
+
+   - Go to **View** -> **Command Palette**
+   - Select **Q#: Install project templates**
+
+1. Open an existing application in VS Code
+
+   - Edit the .csproj file to add the new package versions
```
Comment (cgranade) [inline articles/install-guide/update.md]: If a project depends on the chemistry or numerics packages, those will need to be updated as well in the same way... should we mention that?

File: articles/install-guide/update.md [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+        ```
+
+    - Run the application
+
+        ```bash
+        dotnet run
+        ```
+
+    - Your application will run with the new package versions
+
```
Comment (cgranade) [inline articles/install-guide/update.md]: Do we want to mention the project update instructions here as well?

File: articles/install-guide/update.md (hunk 5 above)
Comment (natke) [reply articles/install-guide/update.md]: I tested this on my own installation, and it didn't happen automatically.  (And I don't think I have opted out of auto updates. Let me check though

Comment (cgranade) [reply articles/install-guide/update.md]: That's very odd, I wonder if the extension hadn't gone live yet?

File: articles/install-guide/update.md (hunk 6 above)
Comment (natke) [reply articles/install-guide/update.md]: I'll add a generic sentence with instructions to also update any other packages

File: articles/install-guide/update.md (hunk 7 above)
Comment (natke) [reply articles/install-guide/update.md]: As discussed offline, the csproj will be updated by the dotnet add package ...

Comment (cgranade) [reply articles/install-guide/update.md]: Never mind, sorry, I had misread your changes. Looks good!

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/14ba10fd2480472c6d376628ca5472240dd0c01d/articles/install-guide/update.md) | 86 | 65 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/98d1d523-67e8-4a80-a4cc-3f80cda79713_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Pull request merged**